api/src/test/java/marquez/BaseIntegrationTest.java

```diff
@@ -13,6 +21,7 @@
   protected static final String CONFIG_FILE = "config.test.yml";
   protected static final String CONFIG_FILE_PATH = ResourceHelpers.resourceFilePath(CONFIG_FILE);
   protected static final PostgresContainer POSTGRES = createMarquezPostgres();
+  HttpClient http2 = HttpClient.newBuilder().version(Version.HTTP_2).build();
```

Comment:
_Minor_: Would we also want to mark  `http2` as `static final`?

Reply:
imo, package private is okay for test classes since there wouldn't really be an instance where this could be abused. I'll migrate it to the others modifiers for consistency.